src/utilities/formInputConfigs.js

```diff
@@ -220,6 +220,8 @@ const FORM_INPUT_CONFIGS = seedObject => ({
     options: UIDatabase.objects('LocationType'),
     optionKey: 'description',
     isEditable: true,
+    validator: input => !!input,
```

Comment:
made location type required. Previously drop downs were assuming there were options and there were no "NONE" option, so it didn't matter